docs/user-guide/horizontal-pod-autoscaling/walkthrough.md

```diff
@@ -20,6 +21,12 @@ as Horizontal Pod Autoscaler uses it to collect metrics
 (if you followed [getting started on GCE guide](/docs/getting-started-guides/gce),
 heapster monitoring will be turned-on by default).
 
+In order to follow the sections on specifying multiple metrics, you will need a cluster and kubectl
```

Comment:
"In order to follow the sections on specifying multiple metrics" -> "To specify multiple resource metrics for Horizontal Pod Autoscaler, you must have a Kubernetes cluster and kubectl at version 1.6 or later."